src/View/Helper/FormHelper.php

```diff
@@ -282,7 +265,7 @@ public function setColumnSizes(array $columns): array
      * @param string $what The type of the column (`'label'`, `'input'`, `'error'`).
      * @return string The classes for the size or offset of the specified column.
      */
-    protected function _getColumnClass(string $what): string
+    public function getColumnClass(string $what): string
```

Comment:
Is there any reason for this to be made `public`? And the `_` prefix is CakePHP convention for protected methods, any reason you removed it from most methods?

Reply:
My phpcs was misconfigured in my IDE. Sorry